packages/ui/src/components/Designer/Sidebar/DetailView/index.tsx

```diff
@@ -123,8 +123,8 @@ Check this document: https://pdfme.com/docs/custom-schemas`);
       key: { title: i18n('fieldName'), type: 'string', required: true, span: 14 },
       '-': { type: 'void', widget: 'Divider' },
       align: { title: i18n('align'), type: 'void', widget: 'AlignWidget' },
-      x: { title: 'X', type: 'number', widget: 'inputNumber', required: true, span: 8 },
-      y: { title: 'Y', type: 'number', widget: 'inputNumber', required: true, span: 8 },
+      x: { title: 'X', type: 'number', widget: 'inputNumber', required: true, span: 8, min: 0 },
```

Comment:
I think we might want to consider schema that start outside of the window. Although I haven't tried this myself so I don't know how it would render 🤔  But I could imagine both text where you want to rotate and position tight to the left edge (negative X pos) or similarly for images where you only want part of the image to be in canvas.

As I say, I haven't tried either of these so I don't know how it would render.